ci/gpu/build.sh

```diff
@@ -119,4 +119,8 @@ else
         py.test --cache-clear -vs `python -c "import distributed.tests.test_nanny as m;print(m.__file__)"`
         py.test --cache-clear -vs `python -c "import distributed.diagnostics.tests.test_nvml as m;print(m.__file__)"`
     fi
+
+    logger "Run local benchmark..."
+    python dask_cuda/benchmarks/local_cudf_shuffle.py --partition-size="1 KiB" -d 0  --runs 1 --backend dask
+    python dask_cuda/benchmarks/local_cudf_shuffle.py --partition-size="1 KiB" -d 0  --runs 1 --backend explicit-comms
```

Comment:
How long does that take? We don't run other benchmarks because they can take too long and aren't really tests. Maybe they would be more suitable for https://github.com/quasiben/dask-cuda-benchmarks ?

Reply:
It only takes 4 second. It is just checking that the benchmark works